closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -23,7 +23,7 @@ include LibCommon.gmk
 ifeq (true,$(BUILD_OPENJCEPLUS))
 
 # Identify the desired JGSKIT target platform.
-OPENJCEPLUS_BOOT_JDK := $(BOOT_JDK)
+OPENJCEPLUS_BOOT_JDK := $(JDK_OUTPUTDIR)
```

Comment:
This can't be expected to work. The JDK in `$(JDK_OUTPUTDIR)` isn't necessarily usable yet.